src/Registration/cReg/tests/test_cReg.cpp

```diff
@@ -1343,6 +1343,35 @@ int main(int argc, char* argv[])
     }
 #endif
 
+#ifdef SIRF_VTK
+    {
+        std::cout << "// ----------------------------------------------------------------------- //\n";
+        std::cout << "//                  Starting VTK test...\n";
+        std::cout << "//------------------------------------------------------------------------ //\n";
+
+        // Get inverse using NR functionality
+        const auto nr_disp_fwd = dynamic_cast<const NiftiImageData3DDisplacement<float>&>(*NA.get_displacement_field_forward_sptr());
+        const auto nr_def_fwd  = NiftiImageData3DDeformation<float>(nr_disp_fwd);
+        const auto nr_def_inv  = nr_def_fwd.get_inverse();
+        const auto nr_disp_inv = NiftiImageData3DDisplacement<float>(*nr_def_inv);
+
+        // Get inverse using VTK
+        auto vtk_disp_inv = nr_disp_fwd.get_inverse_vtk();
+
+        nr_disp_inv.write("/Users/rich/Desktop/nr_disp_inv");
+        vtk_disp_inv.write("/Users/rich/Desktop/vtk_disp_inv");
```

Comment:
obviously will have to change this. Not sure what we're using as a strategy for  temporary files

Reply:
yes, i'll just delete. only needed for debugging.